### ISSUE TITLE
Support mkdocs-material theme toggle

### ISSUE THREAD

**Issue by corykinney** [danielfrg/mkdocs-jupyter#59]
The README describes how to implement the Jupyter Lab dark theme for use with the material slate theme:

``` yaml
plugins:
  - mkdocs-jupyter:
      theme: dark

theme:
  name: material
  palette:
    scheme: slate
```

Is it possible to support toggling between Jupyter Lab themes when material toggles between themes? This is the implementation of the theme toggle in material:

``` yaml
theme:
  palette: 
    - scheme: default
      toggle:
        icon: material/toggle-switch-off-outline
        name: Switch to dark mode
    - scheme: slate 
      toggle:
        icon: material/toggle-switch
        name: Switch to light mode
```

I'm not sure if there are any inherent limitations, but I thought this would be an interesting feature request!

**Comment by danielfrg** [comment]
It could be possible (and a nice to add eventually) with CSS classes or something like that but not something it can do right now.

**Comment by danielfrg** [comment]
This is solved in version `0.20.0`.
